README.Rmd

```diff
@@ -38,6 +45,23 @@ r_oldrel()
 r_versions()
 ```
 
+
+## Use cases in the wild
```

Comment:
Why do we want these? These links are bound to go out of date, no?

Reply:
right but it's interesting for people landing on the docs, and the README is .Rbuildignored :-)

Reply:
we can un-ignore the readme, but I don't think we need "usage in the wild"